backend/plan/urls.py

```diff
@@ -12,5 +17,7 @@
 urlpatterns = [
     path("", TemplateView.as_view(template_name="plan/build/index.html")),
     path("recommendations/", recommend_courses_view, name="recommend-courses"),
+    path("shared/get_schedule", get_shared_schedule, name="get-shared-schedule"),
```

Comment:
A more RESTful route would be GET "schedules/shared" (and we could accept "user_id" as a query parameter to optionally filter shared schedules by a specific friend user -- otherwise we can return all schedules shared with the current user).

Including "get" in the route name is kind of redundant because the method name is GET.

Reply:
I like this design a lot! Will be implementing this, thanks for the suggestion!